specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-02-01/managedClusters.json

```diff
@@ -0,0 +1,5726 @@
+          "description": "This cannot be specified unless the scaleSetPriority is 'Spot'. If not specified, the default is 'Delete'."
+        },
+        "spotMaxPrice": {
+          "$ref": "#/definitions/SpotMaxPrice",
+          "title": "The max price (in US Dollars) you are willing to pay for spot instances. Possible values are any decimal value greater than zero or -1 which indicates default price to be up-to on-demand.",
+          "description": "Possible values are any decimal value greater than zero or -1 which indicates the willingness to pay any on-demand price. For more details on spot pricing, see [spot VMs pricing](https://docs.microsoft.com/azure/virtual-machines/spot-vms#pricing)"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The tags to be persisted on the agent pool virtual machine scale set."
+        },
+        "nodeLabels": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The node labels to be persisted across all nodes in agent pool."
+        },
+        "nodeTaints": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The taints added to new nodes during node pool create and scale. For example, key=value:NoSchedule."
+        },
+        "proximityPlacementGroupID": {
+          "$ref": "#/definitions/ProximityPlacementGroupID",
+          "description": "The ID for Proximity Placement Group."
+        },
+        "kubeletConfig": {
+          "$ref": "#/definitions/KubeletConfig",
+          "description": "The Kubelet configuration on the agent pool nodes."
+        },
+        "linuxOSConfig": {
+          "$ref": "#/definitions/LinuxOSConfig",
+          "description": "The OS configuration of Linux agent nodes."
+        },
+        "enableEncryptionAtHost": {
+          "type": "boolean",
+          "title": "Whether to enable host based OS and data drive encryption.",
+          "description": "This is only supported on certain VM sizes and in certain Azure regions. For more information, see: https://docs.microsoft.com/azure/aks/enable-host-encryption"
+        },
+        "enableUltraSSD": {
+          "type": "boolean",
+          "description": "Whether to enable UltraSSD"
+        },
+        "enableFIPS": {
+          "type": "boolean",
+          "title": "Whether to use a FIPS-enabled OS.",
+          "description": "See [Add a FIPS-enabled node pool](https://docs.microsoft.com/azure/aks/use-multiple-node-pools#add-a-fips-enabled-node-pool-preview) for more details."
+        },
+        "gpuInstanceProfile": {
+          "$ref": "#/definitions/GPUInstanceProfile",
+          "description": "GPUInstanceProfile to be used to specify GPU MIG instance profile for supported GPU VM SKU."
+        },
+        "creationData": {
```

Comment:
There are a number of removals between 2022-02-02-preview (which already exists) and this stable version.  managedClusterSnapshots, some properties in this model, etc...  Is that intentional?

Reply:
its odd but not that bad since 2022-02-02-preview is technically a "later" date than this version

Reply:
Yes, compared to the preview api, the removals in the stable api are intentional. Or more intuitively, we added some additional features (which are in preview or experimental) to the preview api only.

Reply:
As for the strangeness of the date, I manually control the stable api to be merged into the main branch later, so that the default tag in the main branch is the latest stable api.